app.ts

```diff
@@ -10,7 +10,7 @@ import { Application, Request } from 'express'
 const app: Application = express()
 const port = process.env.EXPRESS_PORT || 8160
 
-const detectDevice = (request: Request) => {
+const detectUserDevice = (request: Request) => {
```

Comment:
Generally, come back to this and apply SOLID and various other clean code principles to a) make this easier to work with and b) an exercise in writing cleaner OOP in TypeScript